bot/cogs/auth.py

```diff
@@ -13,12 +15,12 @@ def __init__(self, bot):
     @commands.command(brief='Link a player on the backend')
     async def link(self, ctx):
         """ Link a player by sending them a link to sign in with steam on the backend. """
-        is_linked = await self.bot.api.is_linked(ctx.author.id)
+        user = User(ctx.author.id)
 
-        if is_linked:
+        if await user.is_linked():
             title = f'Unable to link **{ctx.author.display_name}**: They are already linked'
         else:
-            link = await self.bot.api.generate_link_url(ctx.author.id)
+            link = await user.generate_link_url(ctx.author.id)
```

Comment:
This function now takes 0 args so you can remove the `ctx.author.id`.

Reply:
Yeah, missed it